src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -18,15 +18,16 @@
 import java.awt.BorderLayout;
 
 import java.io.IOException;
-
+import java.lang.reflect.Array;
```

Comment:
I am pretty sure you do not need the reflection API in your code. Maybe you imported the wrong class?